### PR TITLE
Add EIP: No-Ether transactions with free-for-all tips

### PR DESCRIPTION
**Pull request by fulldecent**
Discussing a new approach for third-party pay transactions

### CODE REVIEW

File: EIPS/eip-8888.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+eip: 8888
+title: No-Ether transactions with free-for-all tips
+description: Externally-owned account having no Ether can send transactions and pay tips using a new "free-for-all" bucket
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-8888-no-ether-transactions-with-free-for-all-tips/21108
```
Comment (abcoathup) [inline EIPS/eip-8888.md:6]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7768-no-ether-transactions-with-free-for-all-tips/21108
```
I updated the topic title in Eth Magicians to use the assigned EIP number.

File: EIPS/eip-7768.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-7768-no-ether-transactions-with-free-for-all-tips/21108
+status: Draft
+type: Meta
+created: 2024-09-14
+---
+
+## Abstract
+
+A technique is introduced where an externally-owned account having no Ether can send transactions and pay tips using a new "free-for-all" bucket and using their own `origin.tx`. This requires no client changes and is compatible with existing ecosystem parts.
```
Comment (SamWilsn) [inline EIPS/eip-7768.md:14]: You should provide a high-level (but still technical) overview here of how this works. For example, what's a bucket? How does the block producer get paid?

File: EIPS/eip-7768.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+## Motivation
+
+There is much interest in third-party-pay transactions on Ethereum and competing networks.
+
+Other proposals require changes to the Ethereum client, that transactions be sent to the network (i.e. `tx.origin`) using a separate account and/or other additional things.
+
+In contrast, this proposal introduces and standardizes a solution to this problem that works only with existing client and technology, and which preserves the `tx.origin` of the originator of a transaction.
```
[the remark this answers is not in the document]
Comment (fulldecent) [reply EIPS/eip-7768.md:22]: I do see that Pectra  / 7702 is included in  https://eips.ethereum.org/EIPS/eip-7600

However 7600 has not reached last call status. And therefore it is still in negotiation.

If Pectra in fact has already been decided, perhaps what you are saying here, then that is a violation of the community process we have set up here.

**Comment by fulldecent** [comment]
Please merge to draft status for a draft

**Comment by fulldecent** [comment]
👋  hello, requesting a draft merge

**Comment by fulldecent** [comment]
Hiyo, merge please?

**Review by xinbenlv — APPROVED** [review]
LGTM to merge as a draft status

File: EIPS/eip-7768.md (hunk 2 above)
Comment (fulldecent) [reply EIPS/eip-7768.md:14]: Thank you, updated at https://github.com/fulldecent/EIPs/blob/a2dbd017621aa223e6be669bf3ccae8475938cbb/EIPS/eip-7768.md

**Pull request merged**